module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -180,6 +189,8 @@ public function setConfig($settings)
             ?? $this->useQuery;
         $this->useQueryMinLength = $databasesConfig['useQueryMinLength']
             ?? $this->useQueryMinLength;
+        $this->useQueryReplacePattern = $databasesConfig['useQueryReplacePattern']
+            ?? $this->useQueryReplacePattern;
```

Comment:
Maybe do a little extra analysis to ensure we end up with a string type?

```suggestion
        $queryReplaceConfig = $databasesConfig['useQueryReplacePattern'] ?? $this->useQueryReplacePattern;
        $this->useQueryReplacePattern = $queryReplaceConfig ?: '';
```